src/Command/UpdateCdnStatusCommand.php

```diff
@@ -74,47 +92,51 @@ public function execute(InputInterface $input, OutputInterface $output)
             'cdnIsFlushable' => true,
         ]);
 
-        $this->log(sprintf('Loaded %s medias for updating CDN status (provider: %s, context: %s)', \count($medias), $provider->getName(), $context));
+        $this->log(sprintf('Loaded %s media for CDN status update (provider: %s, context: %s)', \count($medias), $provider->getName(), $context));
 
         foreach ($medias as $media) {
             $cdn = $provider->getCdn();
 
-            $this->log(sprintf('Refresh CDN status for media "%s" (%d) ', $media->getName(), $media->getId()), false);
+            $this->log(sprintf('Refresh CDN status for media "%s" (%s) ', $media->getName(), $media->getId()), false);
 
             if (!$media->getCdnFlushIdentifier()) {
-                $this->log('<error>Skiping while empty flush identifier</error>');
+                $this->log('<error>Skiping since the medium has not a pending flush.</error>');
 
                 continue;
             }
 
+            $previousStatus = $media->getCdnStatus();
+
             try {
-                $previousStatus = $media->getCdnStatus();
-                if (CDNInterface::STATUS_OK === ($cdnStatus = $cdn->getFlushStatus($media->getCdnFlushIdentifier()))) {
-                    $media->setCdnIsFlushable(false);
+                $cdnStatus = $cdn->getFlushStatus($media->getCdnFlushIdentifier());
+                if (\in_array($cdnStatus, [CDNInterface::STATUS_OK, CDNInterface::STATUS_ERROR], true)) {
                     $media->setCdnFlushIdentifier(null);
-                    $media->setCdnFlushAt(new \DateTime());
+
+                    if (CDNInterface::STATUS_OK === $cdnStatus) {
+                        $media->setCdnFlushAt(new \DateTime());
+                    }
                 }
                 $media->setCdnStatus($cdnStatus);
 
                 if (OutputInterface::VERBOSITY_VERBOSE <= $this->output->getVerbosity()) {
                     if ($previousStatus === $cdnStatus) {
-                        $this->log(sprintf('No changes (%d)', $cdnStatus));
+                        $this->log(sprintf('No changes (%u)', $cdnStatus));
```

Comment:
TIL :bulb:  Makes me wonder why I ever used `%d`

Reply:
I think because in some cases, negative integers should be accepted.

Reply:
yes but that's a really rare occurrence